Description: Adds the missing maven-core dependency since it's no longer provided by maven-plugin-testing-harness 2.x
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/pom.xml
+++ b/pom.xml
@@ -80,6 +80,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-core</artifactId>
+      <version>3.x</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
       <version>4.12</version>
